.github/workflows/pr.yaml

```diff
@@ -86,14 +86,14 @@ jobs:
 
     steps:
 
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     - name: Login to DockerHub
-      uses: docker/login-action@v1
+      uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKER_USER }}
         password: ${{ secrets.DOCKER_PASS }}
 
     - name: Build documentation
-      run: 'EXCLUDE_TESTING=1 EXCLUDE_DEPLOY=1 NO_PUSH=1 BRANCH_NAME=${{ github.event.pull_request.head.ref }} ./ci.sh'
+      run: 'EXCLUDE_TESTING=1 EXCLUDE_DOCUMENTATION=1 EXCLUDE_DEPLOY=1 NO_PUSH=1 BRANCH_NAME=${{ github.event.pull_request.head.ref }} ./ci.sh'
```

Comment:
Does this rule exclude building documentation for a docs build? shouldn't you just remove the rule itself then?

Reply:
The idea was to disable it first to see if CI gets affected in any adverse way. It would eventually be removed. This was done when i was doing this on and off. I can get it properly cleaned.